### ISSUE TITLE
Style breadcrumbs to look more like Balsamiq Mockup

### ISSUE THREAD

**Issue by wsmith-nwfsc** [nwfsc-fram/boatnet#207]

**Comment by wsmith-nwfsc** [comment]
Looks excellent. We'll want to fix the slight diamond pattern effect.
![image](https://user-images.githubusercontent.com/12972097/56511094-92366100-64e0-11e9-9832-2a9523e16537.png)

**Comment by SaOgaz-NOAA** [comment]
@sethgerou-noaa I feel like things don't use the diamond at all anymore right? It's circles?

**Comment by sethgerou-noaa** [comment]
@SaOgaz-NOAA  no, we still use the diamond shape, and it still needs fixing - you see the whole diamond when you hover over the button.

![image](https://user-images.githubusercontent.com/38139361/95141757-dd6d1f80-0726-11eb-9a7f-4d73f5fd68a9.png)
